### PR TITLE
add covjson-reader v0.11.6 with npm auto-update

### LINKED ISSUE #7942: [Request] Add covjson-reader
**Library name:** covjson-reader
**Git repository url:** https://github.com/Reading-eScience-Centre/covjson-reader
**npm package url(optional):** https://www.npmjs.com/package/covjson-reader
**License(s):**
**Official homepage:**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #7942 
@pvnr0082t please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/Reading-eScience-Centre/covjson-reader
 * Official website (optional, not the repository): https://doc.esdoc.org/github.com/reading-escience-centre/covjson-reader/
 * NPM package url (optional): https://www.npmjs.com/package/covjson-reader
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 0
   - Count of watchers: 2
   - Count of forks: 0
 * NPM download stats (optional):
   - Downloads in the last day: 9
   - Downloads in the last week: 771
   - Downloads in the last month: 1873

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7942, cc @neothemachine

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
ping @pvnr0082t

**Comment by pvnr0082t** [comment]
@kennynaoh Please add `keywords` in packaga.json.

**Comment by kennynaoh** [comment]
@pvnr0082t 
I have added the `keywords` part.
Please help me review it again.
Thanks!

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

File: ajax/libs/covjson-reader/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "reader"
+  ],
+  "author": "Maik Riechert",
+  "dependencies": {
+    "covutils": "^0.2.0",
+    "cbor-js": "^0.1.0",
+    "ndarray": "^1.0.18",
+    "request-promise": "^1.0.2"
+  },
+  "homepage": "https://doc.esdoc.org/github.com/reading-escience-centre/covjson-reader/",
```
Comment (PeterDaveHello) [inline ajax/libs/covjson-reader/package.json]: Did you add this by yourself? I don't think the api document === homepage of a project.

Comment (kennynaoh) [inline ajax/libs/covjson-reader/package.json]: Yes, I will remove it.

**Comment by kennynaoh** [comment]
@pvnr0082t 
I have updated the `homepage` part.
Please help me review it again.
Thanks!

**Pull request merged**